env/ipp_aes_ctr_provider_test.cc

```diff
@@ -0,0 +1,172 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
Can these tests be built into the env_encryption_test as a different parameterized provider now?